lib/tooling_invoker/configuration.rb

```diff
@@ -26,6 +26,10 @@ def containers_dir
       end
     end
 
+    def job_polling_delay
+      (ENV["JOB_POLLING_DELAY"] || 1).to_i
```

Comment:
This will actually be `0.05` in production eventually, so let's use `to_f`

```suggestion
      (ENV["JOB_POLLING_DELAY"] || 1).to_f
```